src/toolkits/object_detection/one_shot_object_detection/util/parameter_sampler.cpp

```diff
@@ -22,6 +22,14 @@ ParameterSampler::ParameterSampler(size_t width, size_t height, size_t dx,
 
 double deg_to_rad(double angle) { return angle * M_PI / 180.0; }
 
+std::vector<double> vector_deg_to_rad(std::vector<double> means_in_degrees) {
```

Comment:
Where possible, get used to passing in parameters using `const type&` instead of type.  Passing in `std::vector<double>` triggers a copy and memory allocation.  It's not noticeably expensive in each operation, but if it happens across a code base it can make it a lot slower.

Reply:
Done.